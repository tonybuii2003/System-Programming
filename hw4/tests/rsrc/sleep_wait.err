[00000.000000] STARTUP
[00000.000027] PROMPT
[00000.100215] INPUT run echo hello world
[00000.100397] CHANGE 366641: none -> running
[00000.100487] PROMPT
[00000.100993] SIGNAL 17
[00000.101019] CHANGE 366641: running -> stopped
[00000.200714] INPUT run sleep 1
[00000.200934] CHANGE 366642: none -> running
[00000.201017] PROMPT
[00000.201634] SIGNAL 17
[00000.201668] CHANGE 366642: running -> stopped
[00000.301226] INPUT cont 1
[00000.301256] CHANGE 366642: stopped -> running
[00000.301263] PROMPT
[00000.401366] INPUT cont 0
[00000.401403] CHANGE 366641: stopped -> running
[00000.401420] PROMPT
hello world
[00000.402361] SIGNAL 17
[00000.402396] CHANGE 366641: running -> dead
[00000.501660] INPUT wait 1
[00001.302016] SIGNAL 17
[00001.302067] CHANGE 366642: running -> dead
[00001.302091] PROMPT
[00001.402157] INPUT quit
[00001.402175] SHUTDOWN
