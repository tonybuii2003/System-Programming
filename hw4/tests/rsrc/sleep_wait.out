0	366641	T	running		echo hello world
0	366641	T	stopped		echo hello world
1	366642	T	running		sleep 1
1	366642	T	stopped		sleep 1
1	366642	T	running		sleep 1
0	366641	T	running		echo hello world
0	366641	T	dead	0x0	echo hello world
1	366642	T	dead	0x0	sleep 1
